docker/torch/pkg_install.sh

```diff
@@ -87,11 +102,21 @@ build_rock pkg/image \
 build_rock pkg/optim \
            optim-1.0.5-0.rockspec
 
+# CUDA
+build_rock extra/cutorch \
+           rocks/cutorch-scm-1.rockspec
+build_rock extra/cunn \
+           rocks/cunn-scm-1.rockspec
+build_rock extra/cudnn \
```

Comment:
Please use the [R7 branch](https://github.com/soumith/cudnn.torch/tree/R7) for `cudnn`.